docs/getting_started/ai-resources.md

```diff
@@ -0,0 +1,73 @@
+# AI Trainings and Resources Guide
+
+## CU Boulder & CURC Resources
+
+### CU Boulder: AI Efforts on Campus
+- Overview of AI initiatives and support across campus. [https://www.colorado.edu/information-technology/ai-cu-boulder/ai-efforts-campus](https://www.colorado.edu/information-technology/ai-cu-boulder/ai-efforts-campus)
+### RMACC-CU AI Workshop
+- Workshops held in partnership with the Rocky Mountain Advanced Computing Consortium. [https://www.colorado.edu/rc/cu-rmacc-ai-workshop](https://www.colorado.edu/rc/cu-rmacc-ai-workshop)
+### CURC AI/ML Trainings
+- CURC offers trainings each semester to help researchers deploy and optimize AI/ML workflows.
+    * [Setting up LLMs on CURC Resources (an RC Short Course)](https://github.com/ResearchComputing/setting_up_llms_on_curc_resources_rc_short_course)
+    * [Applied Containerization for Machine Learning in HPC (an RC Short Course)](https://colorado.libcal.com/calendar/events/contain)
```

Comment:
This link will eventually be outdated, I think. So, I would suggest that you put an AC in your story for this training that includes changing this link to the GitHub.